analysis/analyze_gor.py

```diff
@@ -69,10 +68,9 @@ def process_game(self, game: GameRecord) -> GorAnalytics:
         return GorAnalytics(
             skipped=False,
             game=game,
-            expected_win_rate=black.with_handicap(get_handicap_adjustment(white.rating, game.handicap,
+            expected_win_rate=black.with_handicap(get_handicap_adjustment("black", black.rating, game.handicap,
```

Comment:
This is the drive-by code change. Now it matches the first call to `gor_update()` above, which I'm 99.9% is the intent (since every either file does this).